src/store/slices/filters/filters.slice.ts

```diff
@@ -0,0 +1,23 @@
+import {PayloadAction, createSlice} from '@reduxjs/toolkit';
+import {FiltersState, Filters} from './filters.model';
+
+const initialState: FiltersState = {
+  filters: {
+    priceRange: [1, 1000],
```

Comment:
It will be better to have separate constants for min and max value and then use them here.